scenarios/aws/fakeintake/params.go

```diff
@@ -28,15 +26,6 @@ func WithoutLoadBalancer() Option {
 	}
 }
 
-// WithName sets the name of the fakeintake.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why was it removed ? We need it for the fallback test

Reply:
Because `Name` is now standard parameter for all components given at creation.